priv/repo/migrations/20170607142123_create_setting.exs

```diff
@@ -0,0 +1,17 @@
+defmodule CoursePlanner.Repo.Migrations.CreateSetting do
+  use Ecto.Migration
+
+  def change do
+    create table(:settings) do
```

Comment:
I don't remember where I read it but it wasn't a concrete example.
You can start by checking the `:map` type in ecto.